spec/features/families/admin_views_all_active_families_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+feature "Admin can see all the active families when in families#index" do
+  scenario "successfully" do
+    visit families_path
+
+    expect(page).to have_text(I18n.t("families.index.active_families"))
```

Comment:
This spec is missing actually seeing a family on the page.
I would add an active family and an inactive family, and check which one is displayed on the page

Reply:
I created traits for all_true_families and all_false_families to test it properly because I have true/false also for bus_drivers and prof. I modified all 3 specs to use it, done in sha `b6c4d17`